test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -221,6 +221,44 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 					Expect(err).NotTo(HaveOccurred())
 					Expect(pass).To(BeTrue())
 				}
+
+				err = iisDeploy.Delete()
+				Expect(err).NotTo(HaveOccurred())
+			} else {
+				Skip("No windows agent was provisioned for this Cluster Definition")
+			}
+		})
+	})
+
+	Describe("with a windows agent pool", func() {
```

Comment:
you dont need to duplicate the describe block here. Instead, just put this `It("should..."` block in the other windows pool `Describe`

Reply:
sure. updated.